installer/bundle/bundle_skel.sh

```diff
@@ -550,10 +549,10 @@ do
             ;;
 
         --collectd)
-            if [ -d "${COLLECTD_DIR}" ]; then
+            if [ -f /etc/collectd.conf ] || [ -f /etc/collectd/collectd.conf ]; then
```

Comment:
In shell, that's not the way you do that.

This should be something like:

`if [ -f /etc/collectd.conf  -o -f /etc/collectd/collectd.conf ]; then`

I'm not even sure the || construct will work reliably, as that's a pipe thing in this context.